### PR TITLE
[ADD] ir_rule_website: initial

### PR DESCRIPTION
**Pull request by iledarn**

### CODE REVIEW

File: ir_rule_website/tests/test_compute_domain.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+    def test_base(self):
+        IrRule = self.env['ir.rule']
+        rule = IrRule.create({'name': 'test ir_rule_website',
+                              'model_id': self.env.ref('base.model_res_partner').id,
+                              'domain_force': "[('parent_id', 'in', [website_id])]"})
+        demo_user = self.env.ref('base.group_system')
+        website_id = 1
+        test_domain = ('parent_id', 'in', [website_id])
+        domain = IrRule.sudo(user=demo_user.id).with_context(website_id=website_id)._compute_domain('res.partner')
+        self.assertTrue(test_domain in domain)
```
Comment (yelizariev) [inline ir_rule_website/tests/test_compute_domain.py]: What you need to check, is that ``_compute_domain`` cache depending on website

File: ir_rule_website/tests/test_compute_domain.py [hunk 2]
```diff
@@ -0,0 +1,18 @@
+# -*- coding: utf-8 -*-
+from openerp.tests.common import TransactionCase
+
+
+class TestComputeDomain(TransactionCase):
+    at_install = True
+    post_install = True
+
+    def test_base(self):
```
Comment (yelizariev) [inline ir_rule_website/tests/test_compute_domain.py]: rename to test_cache

File: ir_rule_website/tests/test_compute_domain.py [hunk 3]
```diff
@@ -6,13 +6,17 @@ class TestComputeDomain(TransactionCase):
...
         demo_user = self.env.ref('base.group_system')
-        website_id = 1
-        test_domain = ('parent_id', 'in', [website_id])
-        domain = IrRule.sudo(user=demo_user.id).with_context(website_id=website_id)._compute_domain('res.partner')
-        self.assertTrue(test_domain in domain)
+
+        def test_cache_compute_domain(website_id):
+            test_domain = ('parent_id', 'in', [website_id])
+            domain = IrRule.sudo(user=demo_user.id).with_context(website_id=website_id)._compute_domain('res.partner')
+            self.assertTrue(test_domain in domain)
```
Comment (yelizariev) [inline ir_rule_website/tests/test_compute_domain.py]: Move this method to class level

**Comment by yelizariev** [comment]
Fix lints to prepare it for publication
```

r_rule_website/models/ir_rule.py:3:1: F401 'odoo.fields' imported but unused
ir_rule_website/tests/test_compute_domain.py:11:39: E201 whitespace after '('
ir_rule_website/__init__.py:1: [E7995(absent-icon), ] File: static/description/icon.png is absent in module. Get possible icons here: https://odoo-development.readthedocs.io/en/latest/dev/docs/icon.png.html
```

**Comment by iledarn** [comment]
@yelizariev , done

**Pull request merged**